### PR TITLE
SAT: test strictness level migration tooling (create issues script)

### PR DESCRIPTION
**Pull request by alafanechere**
## What
Addition of scripts to create a GitHub issue for each GA connector we want to migrate to `high` test strictness level.

This is [step 4 of our execution plan](https://github.com/airbytehq/airbyte/pull/19020).

## How
Please check the README.md file:
https://github.com/airbytehq/airbyte/blob/389bd86cb77a8678bfdd5d49655947988384be5c/airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/README.md#L1

[Example of an issue created with this script](https://github.com/airbytehq/airbyte/issues/19018)

Output of the `create_issues.py ` script in `--dry-run`: 

```bash
INFO:root:[DRY RUN]: gh issue create --title Source AlloyDB for PostgreSQL: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpz1ms5j5r --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Amazon Ads: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp9moz2eaf --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Amplitude: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpxq2_o62_ --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Bing Ads: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpi0ok1vip --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Chargebee: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmps_7cwzbs --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
...
```

### CODE REVIEW

File: airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+3. Find and already existing issue with the same title.
+4. Create the issue and return its url if it does not exist.
+
+Issues get created with the following labels:
+* `area/connectors`
+* `team/connectors-python` 
+* `type/enhancement` 
+* `test-strictness-level`
+
+Issues are added to the following project: `SAT-high-test-strictness-level`
```
Comment (pedroslopez) [inline airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/README.md:28]: Nice- I'm guessing this was easier to do than adding to a Zenhub epic?

File: airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/create_issues.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import logging
+import os
+import subprocess
+import tempfile
+
+from definitions import GA_DEFINITIONS
+from jinja2 import Environment, FileSystemLoader
+
+TEMPLATES_FOLDER = "./templates/"
+COMMON_ISSUE_LABELS = ["area/connectors", "team/connectors-python", "type/enhancement", "test-strictness-level"]
```
Comment (pedroslopez) [inline airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/create_issues.py:17]: Note that we also have connector-specific labels (e.g. `connectors/source/youtube-analytics`) - we _may_ be able to infer these labels from the docker image name but I don't know if we're always following that pattern, so for these purposes it's probably ok to leave out.

**Review by pedroslopez — APPROVED** [review]
Sweet! Could you post the dryrun output or just how many issues we'll create?

File: airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/README.md (hunk 1 above)
Comment (alafanechere) [reply airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/README.md:28]: Yes :) I'll add to a Zenhub epic manually afterward, but it's a good way of quickly finding everything created by this script.

File: airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/create_issues.py (hunk 2 above)
Comment (alafanechere) [reply airbyte-integrations/bases/source-acceptance-test/tools/strictness_level_migration/create_issues.py:17]: Yes I was able to infer the labels quite right, but I'm afraid some of these are not matching perfectly the usual naming rule. If a label does not exist the CLI fails. And there's no feature to list existing labels with the CLI.
So I thought it was good enough not to add the connector-specific label.

**Comment by alafanechere** [comment]
Thanks for your review Pedro! 41 issues will be created:
```
INFO:root:[DRY RUN]: gh issue create --title Source AlloyDB for PostgreSQL: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpw5m4ko_4 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Amazon Ads: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpoc9_abxy --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Amplitude: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpj5wmhxup --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Bing Ads: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp82hgvxyj --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Chargebee: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpa1983pvu --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Facebook Marketing: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp16o45x1r --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source File: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp9fyjv3zv --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Freshdesk: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpcq520bx3 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source GitHub: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmplzvhhzue --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Google Ads: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp2z197any --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Google Analytics (Universal Analytics): enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpnw8v79qh --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Google Search Console: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpvqy4nzq8 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Google Sheets: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmptzu7r214 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Greenhouse: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpxm88rgxd --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Harvest: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp_v9_1yk6 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source HubSpot: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpkbiia1sl --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Instagram: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmprufxeub1 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Intercom: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmphg2jfbni --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Iterable: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp97l2kgk1 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Klaviyo: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpvp4bea1l --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source LinkedIn Ads: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp7jscc0xq --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Mailchimp: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpomkvkmts --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Marketo: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpt9f7wpnq --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Mixpanel: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpvilun3t9 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Notion: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpzcytp5sf --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Paypal Transaction: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpd8te_abi --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Pinterest: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpnt_emr_f --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Postgres: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpf8elu00b --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Recharge: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpv6rt8fa2 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source S3: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp_j_caikj --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Salesforce: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpiagp_y6m --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Slack: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmppeal_t52 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Snapchat Marketing: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp6wqevd5g --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Stripe: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpfcrit1yl --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source SurveyMonkey: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp22olzg11 --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source TikTok Marketing: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpiyd8cr1m --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Twilio: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmp06uo7ku_ --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Zendesk Chat: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpo3roi0hw --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Zendesk Support: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpi37tfo5x --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Zendesk Talk: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmplox1lghy --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:[DRY RUN]: gh issue create --title Source Sentry: enable `high` test strictness level in SAT --body-file /var/folders/48/tgyb8s3579x2y9cwl77yk6ch0000gn/T/tmpyk55y55a --project SAT-high-test-strictness-level --label area/connectors --label team/connectors-python --label type/enhancement --label test-strictness-level
INFO:root:Created 41 new issues.
```

**Pull request merged**